src/Apps/Order/Routes/__tests__/Payment.test.tsx

```diff
@@ -226,8 +226,6 @@ describe("Payment", () => {
     fillAddressForm(paymentRoute, validAddress)
 
     paymentRoute.find(ContinueButton).simulate("click")
-
-    expect.hasAssertions()
```

Comment:
Is this just some unrelated cleanup?

Reply:
Yup, see this commit https://github.com/artsy/reaction/pull/1345/commits/1dec03116e2acce1092f644861245dd8f413bf49

I made our setup expect that every test should have an assertion, a few test bugs came out of that.